apps/content_types/content_types.py

```diff
@@ -195,11 +196,14 @@ def _update_template_fields(self, updates, original):
                 superdesk.get_resource_service('content_templates').patch(template.get('_id'), {'data': data})
 
     def find_one(self, req, **lookup):
+        is_edit = req and 'edit' in req.args
         doc = super().find_one(req, **lookup)
-        if doc and req and 'edit' in req.args:
+        if doc and is_edit:
             prepare_for_edit_content_type(doc)
         if doc:
             clean_doc(doc)
+        if not doc and not is_edit and lookup.get('_id') in ('text', 'picture', 'audio', 'video', 'composite'):
```

Comment:
we can use `DEFAULT_PROFILES` here instead of `('text', 'picture', 'audio', 'video', 'composite')`